src/components/Toolbar/SearchBar.jsx

```diff
@@ -38,15 +40,29 @@ const SearchBar = () => {
 				inputRef.current.focus();
 			}
 		});
+		window.api.recieve('historyReply', (url) => {
+			url = url[0];
+			const { hostname, pathname } = url;
+
+			if (typeof url === 'object') {
+				setInput(hostname + pathname);
+			} else {
+				setInput(url);
+			}
```

Comment:
Hier mss [url] opvangen in de callback

Reply:
Wat bedoel je hier precies mee?

Reply:
In de callback van de `receive` de url[0] er al direct uithaalt. Dus `([url]) => {}`

Reply:
Aha, wist niet dat dit kon op deze manier

Reply:
Ja als je weet dat je hier altijd een array zal ontvangen, kan je perfect kiezen welke index van de array je wilt opvangen om direct te gebruiken